web/src/views/LoadingPage/LoadingPage.tsx

```diff
@@ -1,14 +1,22 @@
 import React from "react";
 import ReactLoading from "react-loading";
 import { Typography, Grid } from "@material-ui/core";
+import { useTheme, useMainColor } from '../../hooks/Theme';
+
+var color = "#000";
+if (useTheme() === "dark") {
```

Comment:
A better way to retrieve the color from the theme is to use `useTheme` from `"@material-ui/core"` within the component. With that variable you can easily access the color that has been set in the material theme instead of retrieving the raw data. Thus, you would not this snippet in every part of the app requiring the color.